PyISY/Nodes/group.py

```diff
@@ -23,7 +23,7 @@ class Group(object):
     status = Property(0)
     hasChildren = False
 
-    def __init__(self, parent, nid, name, members=None, controllers=None):
+    def __init__(self, parent, nid, name, members=None, controllers=None, notes=False):
```

Comment:
So False means we haven't tried to grab the Notes yet, and None means there are no notes.  Is that OK, or how should I handle that?